src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -841,6 +841,26 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!--
+    ============================================================
+                                        NetCorePublishItemsOutputGroup
+
+    Emit an output group containing all files that get published.  This will be consumed by VS installer projects.
+    ============================================================
+    -->
+  <PropertyGroup>
+    <NetCorePublishItemsOutputGroupDependsOn>
+      $(NetCorePublishItemsOutputGroupDependsOn);
+      ComputeFilesToPublish;
+    </NetCorePublishItemsOutputGroupDependsOn>
+  </PropertyGroup>
+
+  <Target Name="NetCorePublishItemsOutputGroup" DependsOnTargets="$(NetCorePublishItemsOutputGroupDependsOn)" Returns="@(NetCorePublishItemsOutputGroupOutputs)">
```

Comment:
I think we should use a name other than NetCore. It is possible to have an SDK project targeting other frameworks and invoke publish and this.

Reply:
ok, how about just "PublishItemsOutputGroup"?

Reply:
Fixed